CONTRIBUTING.md

```diff
@@ -275,6 +275,26 @@ By giving feedback on this work (and related supporting work), you can help us m
 Finally, if nothing brings you more satisfaction than seeing every last issue labeled and all resolved issues closed, feel free to message @cart for a Bevy org role to help us keep things tidy.
 As discussed in [*How we're organized*](#how-were-organized), this role only requires good faith and a basic understanding of our development process.
 
+### How to adopt pull requests
+
+Occasionally authors of pull requests get busy or become unresponsive, in other hand, on the project side, sometimes members fail to reply in a timely manner.
```

Comment:
Nit: Minor phrasing suggestion
```suggestion
Occasionally authors of pull requests get busy, become unresponsive, or simply do not receive review/approval in a timely manner.
```

(may want to double-check with @mockersf, though)

Reply:
your suggestion looks good!

Reply:
Done @mockersf and @MrGVSV.